flint/interface.py

```diff
@@ -33,6 +33,10 @@ def html_to_rdl(html: str) -> str:
         result = result.replace(html_tag, rdl_tag)
     return result
 
+@cached
+def get_infocardmap() -> dict:
```

Comment:
I know the file has different casing but `get_infocard_map` would be a better name imo